lib/system_description.rb

```diff
@@ -265,6 +265,16 @@ def validate_file_data!
     end
   end
 
+  def latest_update
+    attributes.keys.map { |scope| self[scope].meta.try(:[], "modified") }
+      .compact.map { |t| Time.parse(t) }.sort.last
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Do not use `Time.parse` without zone. Use one of `Time.zone.parse`, `Time.parse.current`, `Time.parse.in_time_zone`, `Time.parse.utc`, `Time.parse.getlocal`, `Time.parse.iso8601`, `Time.parse.jisx0301`, `Time.parse.rfc3339`, `Time.parse.to_i`, `Time.parse.to_f` instead.